File: 800/Stones on the Table/stones.py
# https://codeforces.com/problemset/problem/1742/A

n = int(input("Enter the number of stones: "))
print("Stones are: ", n)
a = []
count = 0
for i in range(0, n):
  l = input()
  a.append(l)

print("The array is: ", a)
for i in range(0, n, 1):
  if a[i] == a[i + 1]:
    count = count + 1

print("Count of stones: ", count)

File: 800/Stones on the Table/stones.c
// https://codeforces.com/problemset/problem/266/A
#include <stdio.h>

// Main driver
int main() {
    char arr[50]; 
    int n, count = 0, i, j;
    printf("Enter the number of stones: ");
    scanf("%d", &n);
    printf("Enter the stones.\n");
    for(i = 0; i < n; i++) {
        printf("Enter the color: ");
        scanf("%s", &arr[i]);
    }

    printf("Stones are: ");
    //for(i = 0; i < n; i++) {
    printf("%s", arr);
   // }
    printf("\n");

    for(i = 0, j = i + 1; i < n, j < n; i++, j++) {
        if(arr[i] == arr[j]) {
            count++;
        }
    }
    printf("Count is: %d", count);
    return 0;
}
